specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/examples/galleryExamples/GalleryApplicationVersion_Create.json

```diff
@@ -48,6 +48,14 @@
               "remove": "del C:\\package "
```

Comment:
Add a description to this PR describing the feature being introduced here.

Reply:
Updated description